go.mod

```diff
@@ -64,7 +64,7 @@ require (
 	github.com/hashicorp/go-kms-wrapping/wrappers/transit/v2 v2.0.12
 	github.com/hashicorp/go-memdb v1.3.4
 	github.com/hashicorp/go-metrics v0.5.4
-	github.com/hashicorp/go-msgpack/v2 v2.1.2
+	github.com/hashicorp/go-msgpack/v2 v2.1.3
```

Comment:
for posterity, the relevant change here is hashicorp/go-msgpack#26, which is theorized to be no-op, but "just in case", yes?